tests/test_linops/test_linops_cases/linear_operator_cases.py

```diff
@@ -69,6 +69,12 @@ def case_identity(n: int) -> Tuple[pn.linops.LinearOperator, np.ndarray]:
     return pn.linops.Identity(shape=n), np.eye(n)
 
 
+@pytest_cases.case(tags=("symmetric"))
```

Comment:
Here, the operator is not symmetric in all cases

Reply:
Oops, sorry! I removed that tag just now